.gitignore

```diff
@@ -20,4 +20,10 @@
 /dist/
 
 # ignore mypy
-*mypy*
\ No newline at end of file
+*mypy*
+
+# Ignore API keys
+config.py
+
+# Jupyter notebooks
+*.ipynb*
```

Comment:
will you be using a jupyter notebook in this project?

Reply:
I was using one earlier when I started mocking out the weather.py file.